backend/question.py

```diff
@@ -6,3 +6,73 @@
 @question_page.route('/question/ping', methods=['GET'])
 def question_ping():
     return "Pong", 200
+
+# allow user to post a question and word account not more than 200 words
+@question_page.route('/question/add', methods=['POST'])
+def question_add():
+    data = request.get_json()
+    user_id = data['user_id']
+    question = data['question']
+    answer = data['answer']
+    if len(question) > 200:
+        return make_response(jsonify({"error": "Question is too long"})), 400
+    if len(answer) > 200:
+        return make_response(jsonify({"error": "Answer is too long"})), 400
+    con = sqlite3.connect(DATABASE_NAME)
+    cur = con.cursor()
+    sql = "insert into questions values (?, ?, ?, ?)"
+    cur.execute(sql, [None, user_id, question, answer])
+    con.commit()
+    return make_response(jsonify({"success": "Question added"})), 200
+
+
+# allow user to get all questions
+@question_page.route('/question/get', methods=['GET'])
+def question_get():
+    con = sqlite3.connect(DATABASE_NAME)
+    cur = con.cursor()
+    sql = "SELECT * from questions"
+    rows = cur.execute(sql).fetchall()
+    return make_response(jsonify(rows)), 200
+
+"""
+# allow user to sort the questions by the number of answers
+@question_page.route('/question/sort', methods=['GET'])
+def question_sort():
+    con = sqlite3.connect(DATABASE_NAME)
+    cur = con.cursor()
+    sql = "SELECT * from questions ORDER BY answers DESC"
+    rows = cur.execute(sql).fetchall()
+    return make_response(jsonify(rows)), 200
+"""
+
+# allow user to follow a question
+@question_page.route('/question/follow', methods=['POST'])
+def question_follow():
```

Comment:
we can do the /question/follow later. Most likely follow will be saved in the users table